sample-my.cnf

```diff
@@ -0,0 +1,8 @@
+[client]
+database = babybuddy
+host = localhost
+user = babybuddy
+password = babybuddy
+default-character-set = utf8
+use_unicode = True
+init_command = 'SET default_storage_engine=INNODB'
```

Comment:
Lets move this file to the `etc` dir in the repo root. This is where configuration stuff is stored.